README.md

```diff
@@ -37,7 +46,12 @@ func main() {
 ## Inline mode
 
 As of January 4, 2016, Telegram added inline mode support for bots.
-Telebot support inline mode in a fancy manner.
+Telebot support inline mode in a fancy manner. Since you are probably
```

Comment:
I can't see how one is coming out of another. It all started with inline modes and there goes the env var thing?